Sources/TuistCore/Cache/CacheCategory.swift

```diff
@@ -1,5 +1,5 @@
 /// A cache category.
-public enum CacheCategory: String, CaseIterable, RawRepresentable {
+public enum CacheCategory: CaseIterable {
```

Comment:
What is this used for, wasn't used only for the clean command?

Reply:
it's still used in other places to find artifacts in the cache.